package.json

```diff
@@ -59,20 +59,15 @@
     "react-dom": "^16.0.0 || ^17.0.0 || ^18.0.0"
   },
   "dependencies": {
-    "@types/d3-interpolate": "^2.0.0",
-    "@types/d3-scale": "^3.0.0",
-    "@types/d3-shape": "^2.0.0",
     "classnames": "^2.2.5",
-    "d3-interpolate": "^2.0.0",
-    "d3-scale": "^3.0.0",
-    "d3-shape": "^2.0.0",
     "eventemitter3": "^4.0.1",
     "lodash": "^4.17.19",
     "react-is": "^16.10.2",
     "react-resize-detector": "^7.1.2",
     "react-smooth": "^2.0.1",
     "recharts-scale": "^0.4.4",
-    "reduce-css-calc": "^2.1.8"
+    "reduce-css-calc": "^2.1.8",
+    "victory-vendor": "^36.6.8"
```

Comment:
Could you please document the context of this workaround and the next steps?
You could write a comment in the package.json right above this line.

Reply:
Imo we might need a better more visible place for documenting some of the technical decisions we're making like this one. Not sure where though

Reply:
Comments are not permitted in `json` files, I will add my documentation steps in the thread instead of the review so they're easier to see